service_info/static/less/base/header/nav.less

```diff
@@ -121,6 +127,27 @@
     }
   }
 
+  .collapsible {
```

Comment:
This ladder of width settings is to ensure that there's no overlap between menu item links and menu dropdown buttons. It's an ugly solution, but the preexisting use of absolute positioning (and if there's a better way to do what needs to be done here, I don't know what it is) makes absolute dimensions a convenient layout solution here.